files_to_review/customQueryGenerator.js

```diff
@@ -100,13 +108,8 @@ function buildCustomQueryFunctionScript(propertySelector, selector){
 					}\
 					for (i=0; i<object.length; i++){\
 						var element = object[i].key["0"];\
```

Comment:
There is some consideration to putting in a general format of the function string but due to the morphing based on selections, that base may not provide any more useful information than simply having a copy of that base commented before the function.